spec/support/matchers.rb

```diff
@@ -94,3 +94,24 @@
     "expected that #{actual.url} would link to valid URLs, but found:\n#{bad_urls.join("\n")}"
   end
 end
+
+RSpec::Matchers.define :be_uniquely_titled do
+  # Attempts to validate conformance to WCAG Success Criteria 2.4.2: Page Titled
+  #
+  # Visiting a page with the default app name title is considered a failure, and should be resolved
+  # by providing a distinct description for the page using the `:title` content block.
```

Comment:
Should consider updating this since it was tailored for the IdP:

```suggestion
  # by providing a distinct description for the page via the `title` YAML front-matter metadata.
```

Reply:
Good catch, thanks!